frontend/apps/reader/readerui.lua

```diff
@@ -290,19 +291,19 @@ function ReaderUI:init()
         end
         -- make sure we render document first before calling any callback
         self:registerPostInitCallback(function()
-            local start_tv = TimeVal:now()
+            local start_time = time.now()
             if not self.document:loadDocument() then
                 self:dealWithLoadDocumentFailure()
             end
-            logger.dbg(string.format("  loading took %.3f seconds", TimeVal:getDuration(start_tv)))
+            logger.dbg(string.format("  loading took %.3f seconds", time.get_duration(start_time)))
```

Comment:
Want to keep the `get_` ? `time.duration(start_time)` or `time.duration_since(start_time)` reads clearer.

Will you add a `time.delay_until(some_futur_time)` - which I guess is just the negation of `duration_since()` ?
Btw, does your fts allows for negative durations ? Seems it does, I see some `time.s(-61)` below.

Reply:
Actually duration seem also wrong. Duration of one point in time? 
`time_since` feels more naturally. (And yes negative fts is no problem (until the year 1970-285)